src/main/java/core/basesyntax/Calculable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Calculable {
+    double calculate(int a, int b, char operand);
```

Comment:
```suggestion
    double calculate(int firstOperand, int secondOperand, char operator);
```
same for implementation